tests/test_matrix_utils.py

```
@@ -6553,3 +6660,266 @@ async def test_handle_detection_sensor_packet_queue_fail():
+        patch("mmrelay.matrix_utils._get_msgs_to_keep_config", return_value=5),
+        patch(
+            "mmrelay.matrix_utils._create_mapping_info",
+            return_value={"matrix_event_id": "$event123"},
+        ) as mock_mapping,
+        patch("mmrelay.matrix_utils.queue_message", return_value=True),
+        patch("mmrelay.matrix_utils.get_message_queue", return_value=mock_queue),
+    ):
+        await on_room_message(room, event)
+
+    mock_mapping.assert_called_once()
+    args, kwargs = mock_mapping.call_args
```

Ruff (0.14.10)

Unpacked variable `kwargs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)